src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java

```diff
@@ -0,0 +1,99 @@
+package org.vandeseer.integrationtest;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.vandeseer.TestUtils.getActualPdfFor;
+import static org.vandeseer.TestUtils.getExpectedPdfFor;
+
+import java.awt.Color;
+import java.io.IOException;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.junit.Test;
+import org.vandeseer.TestUtils;
+import org.vandeseer.easytable.OverflowOnSamePageRepeatableHeaderTableDrawer;
+import org.vandeseer.easytable.structure.Row;
+import org.vandeseer.easytable.structure.Table;
+import org.vandeseer.easytable.structure.cell.TextCell;
+
+import de.redsix.pdfcompare.CompareResult;
+import de.redsix.pdfcompare.PdfComparator;
+
+public class OverflowOnSamePageRepeatableHeaderTableDrawerTest {
```

Comment:
This class doesn't test anything yet.
Interestingly I also can't open the new PDF file.

Essentially it's missing the comparing part, like this:
```
final CompareResult compareResult = new PdfComparator<>(
                getExpectedPdfFor(OVERFLOW_ON_SAME_PAGE_PDF),
                getActualPdfFor(OVERFLOW_ON_SAME_PAGE_PDF)
        ).compare();

        assertTrue(compareResult.isEqual());
```

Reply:
I'm not sure if you mean something else, but I'm comparing the pdfs at line 45 :)

Reply:
Indeed! My bad, I simply overlooked it. Sorry!